gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/modules/liquid_tanks/liquid_features.bolt

```diff
@@ -0,0 +1,25 @@
+from gm4_liquid_tanks:liquid_wrappers import liquid
+
+@liquid
+class BottledLightning:
+    id = 'liab_lightning'
+    name = "Lightning"
+    capacity = 300
+    skin = 'lightning_liquid'
+
+    liquid.init()
+    
+    @liquid.item_fill("glass_bottle", "potion[custom_data~{gm4_lightning_in_a_bottle:1b}]", 1)
+    def fill_bottle():
+        loot replace entity 344d47-4-4-4-f04ce104d weapon.mainhand loot gm4_lightning_in_a_bottle:items/bottle_of_lightning
```

Comment:
This UUID is `#declared` into the old datapack-helper-plus's autocomplete, and should also be working with spyglass if that feature is functional. I think this is pretty much the same as the hard-coded coordinates that show up in lib forceload's blocks (thats where this armor stand is located after all), so I don't think it would actually get that much difference.

Reply:
Okay I didn't realize this was the forceloaded chunk entity, in that case it makes sense to keep it like this.